packages/button/test/lion-button.test.js

```diff
@@ -206,7 +206,7 @@ describe('lion-button', () => {
       const wrapperId = el.getAttribute('aria-labelledby');
       expect(/** @type {ShadowRoot} */ (el.shadowRoot).querySelector(`#${wrapperId}`)).to.exist;
       expect(/** @type {ShadowRoot} */ (el.shadowRoot).querySelector(`#${wrapperId}`)).dom.to.equal(
-        `<div id="${wrapperId}"><slot></slot></div>`,
+        `<div class="slot" id="${wrapperId}"><slot></slot></div>`,
```

Comment:
Maybe we should rename the class it to `button-content` or `btn-content` to be a bit more descriptive.